tests/test_output_render.py

```diff
@@ -79,7 +79,8 @@ class JCase:
     local: str = attr.ib()
     remote: dict = attr.ib()
     output_default: str = attr.ib()
-    output_struct: list = attr.ib()
+    output_yaml: str = attr.ib()
+    output_json: list = attr.ib()
```

Comment:
nah. You'd be better off seeing if the parsed structures from the json/yaml match expectations.